assets/sass/patterns/molecules/site-header-nav-bar-primary.scss

```diff
@@ -51,10 +48,34 @@
   text-decoration: none;
 }
 
+.nav-primary__item--list {
+  @include margin(-35, "left");
```

Comment:
Why do you need that negative margin? I do no see visual difference in the browser.

Reply:
no, i don't need that margin, i removed it